src/components/ModalDrawer.js

```diff
@@ -29,7 +30,7 @@ const ModalDrawer = ({ isOpen, onDismiss, width = 450, children, ...props }) =>
 
   return h(Transition, {
     in: isOpen,
-    timeout: { exit: 200 },
+    timeout: { exit: DEFAULT_TRANSITION_DURATION },
```

Comment:
Using `DEFAULT_TRANSITION_DURATION` doesn't make it clear that the values in the different modals are related and need to be synchronized with `ModalDrawer`. Those could be read as unrelated things that all happen to use the same default value. Instead of a common `DEFAULT_TRANSITION_DURATION` constant, maybe export a constant from this module (`modalDrawerExitTimeout` or something like that) and use that in the consuming components?

Reply:
Or, maybe the need for synchronized timeouts could be removed by updating `ModalDrawer` to allow passing callback props to `Transition` and using [`onExited`](https://reactcommunity.org/react-transition-group/transition#Transition-prop-onExited) to reset views?